python_modules/libraries/dagster-sigma/dagster_sigma/resource.py

```diff
@@ -387,7 +387,7 @@ async def safe_fetch_lineage_for_element(
 
     @cached_method
     async def build_organization_data(
-        self, sigma_filter: Optional[SigmaFilter]
+        self, sigma_filter: Optional[SigmaFilter], skip_fetch_column_data: bool
```

Comment:
I find it a bit strange to have this be defined in the negative -- can it just be `fetch_column_data`?

Reply:
Updated, good call